src/main/java/net/snowflake/client/jdbc/SnowflakeSQLLoggedException.java

```diff
@@ -218,7 +242,13 @@ public SnowflakeSQLLoggedException(
         errorResourceBundleManager.getLocalizedMessage(
             String.valueOf(errorCode.getMessageCode()), params);
     sendTelemetryData(
-        null, reason, errorCode.getSqlState(), errorCode.getMessageCode(), errorCode, session);
+        null,
```

Comment:
Create static function called prior to throwing SQLFeatureNotSupportedException. Since this exception is not actually a SQLException, it cannot be added as a constructor to this class.